client/hotrod-client/pom.xml

```diff
@@ -180,7 +180,6 @@
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
                <systemPropertyVariables combine.self="append">
-                  <io.netty.eventLoopThreads>3</io.netty.eventLoopThreads>
                   <infinispan.server.hotrod.workerThreads>3</infinispan.server.hotrod.workerThreads>
```

Comment:
I don't see this being used

Reply:
This was being used before, but since https://github.com/infinispan/infinispan/commit/c03afca7cc301336de43dc7fa653891a16ad3982 it lost its effect.

Reply:
I replaced this with https://github.com/infinispan/infinispan/pull/6205/files#diff-e61ddaf2e402ece2f579d052e60f8091R208